### ISSUE TITLE
About  "tf.reduce_mean(input_vectors, axis = 1)"

### ISSUE THREAD

**Issue by ZhuYuHe** [nilmagnum/oxford-deep-nlp-2017-solutions#1]
when compute x = (1/N) sum_i xi， I notice that you use tf.reduce_mean(input_vectors, axis = 1).
However, N in the formula is the length of sentence, but tf.reduce_mean compute x use the length of sentence after padding.
So, the correct method to compute x should be sum(x)/len(x).
